servers/rendering/rendering_server_default.h

```diff
@@ -945,10 +945,27 @@ class RenderingServerDefault : public RenderingServer {
 	FUNC0(global_shader_parameters_clear)
 
 #undef server_name
+#undef ServerName
+	/* STATUS INFORMATION */
+#define ServerName RendererUtilities
+#define server_name RSG::utilities
+	FUNC0RC(String, get_video_adapter_name)
+	FUNC0RC(String, get_video_adapter_vendor)
+	FUNC0RC(RenderingDevice::DeviceType, get_video_adapter_type)
```

Comment:
This shouldn't be included here. ``FUNC0RC`` forces functions to be called on the command queue which adds unnecessary performance overhead to functions if they don't need to be called from the rendering thread.

``get_video_adapter_type`` should stay below with the other status information functions